### PR TITLE
Add affiliate cookie name to ecommerce config

### PR DESCRIPTION
**Pull request by bderusha**
FYI @clintonb @peter-fogg

### CODE REVIEW

**Comment by peter-fogg** [comment]
👍

**Comment by clintonb** [comment]
You may need to file a DEVOPS ticket to ensure `EDXAPP_AFFILIATE_COOKIE_NAME` is available globally.

**Comment by clintonb** [comment]
FYI @feanil

**Comment by clintonb** [comment]
👍

**Comment by feanil** [comment]
Yes, we probably want to override this in production so stage and prod don't look at the same cookie for example.

**Comment by clintonb** [comment]
@feanil I hadn't thought of that. Doh!

**Comment by bderusha** [comment]
Ticket created.  https://openedx.atlassian.net/browse/DEVOPS-4216.  We will also need to alter the name of the cookie in GTM for stage

**Comment by clintonb** [comment]
@peter-fogg please update the GTM scripts.

**Comment by peter-fogg** [comment]
I've updated GTM to use `stage_affiliate_id` and opened https://github.com/edx/ecommerce-scripts/pull/6.

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -152,7 +152,7 @@ EDXAPP_API_DOCUMENTATION_URL: "http://edx.readthedocs.org/projects/edx-platform-
 EDXAPP_AUTH_DOCUMENTATION_URL: "http://edx.readthedocs.org/projects/edx-platform-api/en/latest/authentication.html"
 
 # Settings for affiliate cookie tracking
-EDXAPP_AFFILIATE_COOKIE_NAME: 'affiliate_id'
+EDXAPP_AFFILIATE_COOKIE_NAME: 'dev_affiliate_id'
```
Comment (bderusha) [inline playbooks/roles/edxapp/defaults/main.yml:155]: @peter-fogg FYI I've updated this to not conflict with our prod value of `affiliate_id`

**Pull request merged**